doc/source/user_guide/io.rst

```diff
@@ -5464,6 +5464,26 @@ The above example creates a partitioned dataset that may look like:
    except OSError:
        pass
 
+.. note::
+
+   * The parquet representation of ``StringDtype`` is the same, regardless of the storage.
+   * The data will be read in accordance with the ``string_storage`` settings.
+
+.. ipython:: python
+   :okwarning:
```

Comment:
new examples should not warn

Reply:
@phofl I removed the warning but now the build fails with 
`Sphinx parallel build error:
RuntimeError: Non Expected warning in `/home/runner/work/pandas/pandas/doc/source/user_guide/io.rst` line 5485`

This is not throwing any error while building the docs in local. Can you please help me out here?

Reply:
@phofl little help here?